apiserver/debuglog_db_test.go

```diff
@@ -3,16 +3,130 @@
 
 package apiserver_test
 
-import gc "gopkg.in/check.v1"
+import (
+	"bufio"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	jc "github.com/juju/testing/checkers"
+	"github.com/juju/utils"
+	"golang.org/x/net/websocket"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/testing/factory"
+)
 
-// debugLogDBSuite runs the common debuglog API tests when the db-log
-// feature flag is enabled. These tests are inherited from
-// debugLogBaseSuite.
 type debugLogDBSuite struct {
-	debugLogBaseSuite
+	authHTTPSuite
```

Comment:
Thanks for cleaning this up. These reason for this is that /log used to be backed by a file on disk and the suites were split when logging to mongodb was added behind a feature flag. There's not need for the separation now.